client/src/screens/SearchResultPage/SearchResultPage.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react'
+import Navbar from '../../components/SearchResultPage/Navbar/Navbar'
+import classes from './SearchResultPage.module.css'
+import gql from 'graphql-tag'
+import { Query } from 'react-apollo'
+
+export default function SearchResultPage({ match, props }) {
+    const SearchPathway = gql`
+    query{
+    SearchPathways(searchQuery: "${match.params.query}"){
+    name
+    tags{
+        name
+    }
+    }
+}`
+
+    return (
+        <div className={classes.searchResults}>
+            <Navbar />
+            <h3 className={classes.heading}>
+                Displaying search results for "{match.params.query}"
+            </h3>
+
+            <Query query={SearchPathway}>
+                {({ loading, error, data }) => {
+                    if (loading)
+                        return <h3 className={classes.heading}>Loading</h3>
+                    if (error)
+                        return (
+                            <h3 className={classes.heading}>
+                                Error! {error.message}
+                            </h3>
+                        )
+
+                    return data.SearchPathways.map((item) => (
+                        <div className={classes.resultCard}>
+                            <div className={classes.resultCardHeading}>
+                                <a className={classes.link}>{item.name}</a>
```

Comment:
Don't use an anchor tag here. Use the Link tag from react-router-dom. Also, once a pathway is selected, the next page to be rendered has a url of the form  '...../pathway?id={pathwayId}'